lib/json/static.nit

```diff
@@ -31,6 +31,9 @@ private import json_lexer
 interface Jsonable
 	# Encode `self` in JSON.
 	#
+	# This is a recursive method which can be refined by any instances.
+	# To write any `Serializable` object to JSON, see `serialize_to_json`.
+	#
 	# SEE: `append_json`
 	fun to_json: String is abstract
```

Comment:
`to_json` is central API wise, and the long term goal is to deprecate or remove all these services.